libc-bottom-half/headers/public/wasi/libc-find-relpath.h

```diff
@@ -70,7 +70,7 @@ int __wasilibc_find_relpath_alloc(
     char **relative,
     size_t *relative_len,
     int can_realloc
-) __attribute__((weak));
+) __attribute__((__weak__));
```

Comment:
Here is a place that I could have used just `weak` but when I do the build fails:

```
#
# Test that it compiles.
#
clang -O2 -DNDEBUG --target=wasm32-wasi -fno-trapping-math -Wall -Wextra -Werror -Wno-null-pointer-arithmetic -Wno-unused-parameter -Wno-sign-compare -Wno-unused-variable -Wno-unused-function -Wno-ignored-attributes -Wno-missing-braces -Wno-ignored-pragmas -Wno-unused-but-set-variable -Wno-unknown-warning-option -mthread-model single -isystem "/home/abrown/Code/wasi-libc/sysroot/include" -fsyntax-only "/home/abrown/Code/wasi-libc/sysroot/share/wasm32-wasi/include-all.c" -Wno-\#warnings
In file included from /home/abrown/Code/wasi-libc/sysroot/share/wasm32-wasi/include-all.c:166:
/home/abrown/Code/wasi-libc/sysroot/include/wasi/libc-find-relpath.h:73:3: error: expected function body after function declarator
) weak;
  ^
1 error generated.
```

I guess there is some other place to add an include path if we want to use `weak` everywhere. (Not sure I'm fully convinced that `weak` is better than `__attribute((__weak__))`, though... Just trying to be consistent.)

Reply:
Adding `#include <features.h>` should do it.

Reply:
That seemed like the right thing to do so I tried it but it blew up with the same error...